ninja/orm/fields.py

```diff
@@ -175,6 +175,26 @@ def get_schema_field(
             ]
             raise ConfigError("\n".join(msg)) from e
 
+        if hasattr(field, "choices") and field.choices:
+            choices = [choice[0] for choice in field.choices]
+            if null or optional:
```

Comment:
It would be best if we can avoid some of the duplicate code here. You can think of it like we want to funnel everything to the final return statement.

What we should do is replace the `python_type` with a `Literal` for the enum. It's probably the best way to express the enum to Pydantic without the original object.

```suggestion
        if hasattr(field, "choices") and field.choices:
            choices = tuple([choice[0] for choice in field.choices])
            python_type = Literal[choices]
```